src/java/org/apache/cassandra/metrics/ClientRequestsMetrics.java

```diff
@@ -27,8 +27,10 @@
 public class ClientRequestsMetrics
 {
     public final ClientRequestMetrics readMetrics;
+    public final LatencyMetrics readServiceTimeMetrics;
```

Comment:
These new metrics deserve a javadoc. Please highlight how are these different than the existing `readMetrics` and `writeMetrics`. I know the ticket description says it all, but having this repeated here would good for other readers.